inference-engine/tests/functional/inference_engine/cnn_network/cnn_ngraph_impl_tests.cpp

```diff
@@ -125,7 +126,8 @@ TEST(CNNNGraphImplTests, TestGetOutputAfterConvertNetwork) {
     }
 
     InferenceEngine::CNNNetwork cnnNet(ngraph);
-    cnnNet.begin();
+    // TODO
```

Comment:
As I understand the main idea of these tests is a check that we don't loose some information after conversion nGraph to legacy representation.

Before this changes `begin()` call was used for conversion from ngraph to legacy representation.

Reply:
I understand that conversion was triggered, but do we really need  it here?

Reply:
Yes this tests checks that we correctly add output after the conversion. We had a bug on this behavior and I don't want to disable these tests.

Reply:
reverted back such checks.